src/engraving/libmscore/chordrest.cpp

```diff
@@ -917,6 +917,10 @@ EngravingItem* ChordRest::prevArticulationOrLyric(EngravingItem* e)
 
 EngravingItem* ChordRest::nextElement()
 {
+    Tuplet* tuplet = nextChordRest(this)->tuplet();
```

Comment:
Almost there! You still need to null-check the result of `nextChordRest(this)`, before you try to `->tuplet()` on it. You can write like this:
```
ChordRest* nextCR = nextChordRest(this);
Tuplet* = nextCR ? nextCR->tuplet() : nullptr;
if (tuplet) {
    return tuplet;
}
```

Same goes for `prevChordRest(this)` below.